src/task1.js

```diff
@@ -0,0 +1,5 @@
+(() => {
+    process.openStdin().addListener('data', data => { 
+        process.stdout.write('Output' + data.toString().split('').reverse().join('')+ '\n\n');
```

Comment:
Consider creating separate functions for process.stdin.read and process.stdin.write, e.x readFunc and writeFunc, to abstract away these operations.